cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/network/help.xhtml

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Network expansion</title>
+</head>
+<body>
+<div id ="help">
+	<h4>General explanation</h4>
+	<p>
+The network expansion function creates a buffer around the geometry of the source data and applies it to the target data. The example 
+below shows a comparison of the unbuffered source data (left) to the buffered target data (right). <br/>
+
+<img src="../../eu.esdihumboldt.cst.functions.geometric/help/network/NetworkExpansion_Buffer_Map.png" /> <br/>
+
+<strong>Defining the buffer width</strong><br/>
+
+The buffer width can be specified either by a plain value, a mathematical expression or a groovy script.<br>
+</br>
+		<img src="../../eu.esdihumboldt.cst.functions.geometric/help/network/NetworkExpansion_ChoseParameter.png" /> <br/>
+		
+<strong>Plain value</strong><br>
+ </br>
+
+For defining the buffer size using a plain value, enter the value of the desired buffer size (e.g. 500) in the system.<br/>
```

Comment:
To define the buffer size using...

in the function, or in the function dialog?